test/e2e/cluster.sh

```diff
@@ -41,6 +41,17 @@ ${ADD_NODE_POOL_INPUT}
 END
 fi
 
+if [ -n "$PUBLIC_SSH_KEY" ]; then
+  PUBLIC_SSH_KEY=$(cat "${PUBLIC_SSH_KEY}")
```

Comment:
The existing `PUBLIC_SSH_KEY` variable contains the file content, we should not change the meaning of the variable.

If you want to hold the file path, create a new variable `PUBLIC_SSH_KEY_FILE` or similar